packages/terra-core-docs/src/terra-dev-site/doc/cell-grid/About.1.doc.mdx

```diff
@@ -10,8 +11,16 @@ import CellGridPropsTable from 'terra-cell-grid/lib/CellGrid?dev-site-props-tabl
 
 <Badge />
 
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-core/tree/main/packages/terra-cell-grid#readme)
+
 # Terra Cell Grid
 
+<Notice variant="deprecation">
+
+`terra-cell-grid` is no longer supported. If you are using `terra-cell-grid`, consider replacing it with a supported component.
```

Comment:
I feel like "supported component" is vague and we should maybe link alternatives.

Reply:
I agree with adding a specific supported component.

Reply:
Terra Cell Grid is described as "a structural row based layout for aligning columns with 100% of its parents width." I think Terra Table would be the supported component alternative but would appreciate confirmation from @eawww and other reviewers.

Reply:
If we want to point out to only one component, it is `terra-table`. If it's ok to mention more components than one, we can say something like *consider replacing it with one of supported grid components, such as `terra-table`, `terra-worklist-data-grid`, or `terra-flowsheet-data-grid`.*

Reply:
@adoroshk - Cell grid is not really a grid or table but more of a layout mechanism so I hesitate to recommend any of our data grids as alternatives. I would like @eawww  to provide guidance.

Reply:
I agree with @mjpalazzo. Table serves an entirely different purpose. Some other components that do a similar thing to cell grid are `Grid`, `Dynamic Grid`, or, since these most of these don't offer much utility beyond regular ol' CSS Grid or Flexbox, those are options too.